content/millennium/r4/financial/general/charge-item.md

```diff
@@ -157,6 +157,53 @@ List an individual ChargeItem by its id:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Operation: charge-item-credit
+
+<%= beta_tag(action: true) %>
+
+Creates an offsetting ChargeItem for an existing debit ChargeItem.
+
+    POST/ChargeItem/:id/charge-item-credit
+    
```

Comment:
I think we should also provide some implementation notes on the resource, mentioning,
Only the body fields mentioned below are supported. Unsupported fields will be ignored.
and not sure if we want to mention that we need to generate and pass the uuid.  @alexbeckner1

Reply:
Done with only implementation notes. https://github.com/cerner/fhir.cerner.com/pull/726/commits/677935ff04956df418e7d779feb3bd1779522ef0#diff-03f941e4ceb2839bfe0b49a5cc913e04919560e289603ae7b08b875f86ec568dR168-R170